### PR TITLE
removed alpha beta , input is a double value for control

### LINKED ISSUE #71: BrightnessContrast
**Describe the bug**
change from alpha beta to double for controlling brightness contrast

### PR DESCRIPTION
**Pull request by AkshayPS12**
**IMPORTANT: All PRs must be linked to an issue (except for extremely trivial and straightforward changes).**

Fixes #71 

**Checklist**

- [x] I have written Unit Tests
- [x] I have discussed my proposed solution with code owners in the linked issue(s) and we have agreed upon the general approach

### CODE REVIEW

File: base/include/BrightnessContrastControl.h [hunk 1]
```diff
@@ -5,15 +5,16 @@
 class BrightnessContrastControlProps : public ModuleProps
 {
 public:
-	BrightnessContrastControlProps(double _alpha, int _beta) : alpha(_alpha), beta(_beta)
+	BrightnessContrastControlProps(double _contrast, double _brightness, int _bitsPerPixel) : contrast(_contrast), brightness(_brightness), bitsPerPixel(_bitsPerPixel)
```
Comment (kumaakh) [inline base/include/BrightnessContrastControl.h:8]: why should bpp (bits per pixel) be a module prop... it is a Pin Property, The brighness/contrast computation remians the same weather the input is 8bpp, 12bpp or 24bpp, it is not different then the chanel (YUV, RGB, etc.)

File: base/src/BrightnessContrastControl.cpp [hunk 2]
```diff
@@ -117,6 +118,11 @@ void BrightnessContrastControl::setMetadata(framemetadata_sp &metadata)
 	{
 		return;
 	}
+
+	if(mDetail->mProps.bitsPerPixel != 8)
```
Comment (kumaakh) [inline base/src/BrightnessContrastControl.cpp:122]: this check should be done during validating input pins and not here.

File: base/src/BrightnessContrastControl.cpp [hunk 3]
```diff
@@ -153,7 +159,7 @@ BrightnessContrastControlProps BrightnessContrastControl::getProps()
 
 bool BrightnessContrastControl::handlePropsChange(frame_sp &frame)
```
Comment (kumaakh) [inline base/src/BrightnessContrastControl.cpp:160]: referer #64 for the correct name for the module. It should be BrightnessContrastXform. I udnerstand that this was created before we figired out the conventions, please rename the class in this go

**Review by kumaakh — CHANGES_REQUESTED** [review]
add comments, need some minor changes

File: base/src/BrightnessContrastControl.cpp (hunk 3 above)
Comment (AkshayPS12) [reply base/src/BrightnessContrastControl.cpp:165]: I have made the requested changes , please review them

**Review by kumaakh — APPROVED** [review]

**Pull request merged**